components-api/src/main/scala/pl/touk/nussknacker/engine/api/json/decoders/FromJsonSimpleDecoder.scala

```diff
@@ -1,11 +1,11 @@
-package pl.touk.nussknacker.engine.api.json
+package pl.touk.nussknacker.engine.api.json.decoders
 
 import io.circe.Json
 import pl.touk.nussknacker.engine.util.Implicits._
 
 import scala.jdk.CollectionConverters._
 
-object FromJsonDecoder {
+object FromJsonSimpleDecoder {
```

Comment:
I renamed it to `Simple`, because it is a util used by other, more sophisticated decoders or for simple type decoding